examples/example_matrix_multiply.cpp

```diff
@@ -168,7 +168,7 @@ int main() {
               << time.total << " ms\n";
 
     std::cout << "Effective Bandwith:          "
-              << yacx::effective_bandwidth(time.launch, args) << " GB/s\n";
+              << yacx::effective_bandwidth(time.launch, args.size()) << " GB/s\n";
```

Comment:
https://github.com/ZerataX/yacx/blob/master/include/yacx/KernelTime.hpp#L20
these are now best used directly as functions of KernelTime